core/src/Dscp/Core/Arbitrary.hs

```diff
@@ -246,7 +265,8 @@ instance ArbitraryMixture BlockBody
 instance ArbitraryMixture Block
 
 instance Arbitrary ItemDesc where
-    arbitrary = arbitrary `suchThatMap` (rightToMaybe . toItemDesc)
+    arbitrary = (toBase64 @ByteString <$> arbitrary)
```

Comment:
I don't like it, tests should cover any possible scenarious and it would be better to create a separate generator for such smart thing.
I cannot image a case when this change causes us to miss a bug though.

Reply:
Well, type `ItemDesc` assumes that its contents should be somewhat readable, and set of `base64` encoded strings is still quite wide. 
Basically, this change might cause us to miss a bug if we processed the contents of `ItemDesc`s in some way which is more complex than taking a hash of it and putting it into the database, which we do not and I cannot imagine a plausible situation when we will.